modules/memfault-firmware-sdk/config/memfault_metrics_heartbeat_extra.def

```diff
@@ -28,6 +28,7 @@ MEMFAULT_METRICS_KEY_DEFINE(ncs_lte_tx_kilobytes, kMemfaultMetricType_Unsigned)
 MEMFAULT_METRICS_KEY_DEFINE(ncs_lte_rx_kilobytes, kMemfaultMetricType_Unsigned)
 MEMFAULT_METRICS_KEY_DEFINE(ncs_lte_reset_loop_detected_count, kMemfaultMetricType_Unsigned)
 MEMFAULT_METRICS_KEY_DEFINE(ncs_lte_band, kMemfaultMetricType_Unsigned)
+MEMFAULT_METRICS_STRING_KEY_DEFINE(ncs_lte_operator, MODEM_INFO_MAX_SHORT_OP_NAME_SIZE)
```

Comment:
I think this depends on `CONFIG_MODEM_INFO` for the size

Reply:
Done!